web_interface/blueprints/pages_v3.py

```
@@ -306,6 +308,13 @@ def _load_operation_history_partial():
     except Exception as e:
         return f"Error: {str(e)}", 500
 
+def _load_starlark_apps_partial():
+    """Load Starlark apps management partial"""
+    try:
+        return render_template('v3/partials/starlark_apps.html')
+    except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)